src/Broker/Broker.php

```diff
@@ -138,6 +144,7 @@ public function __construct(
 
 		$this->dynamicMethodReturnTypeExtensions = $dynamicMethodReturnTypeExtensions;
 		$this->dynamicStaticMethodReturnTypeExtensions = $dynamicStaticMethodReturnTypeExtensions;
+		$this->operatorTypeSpecifyingExtensions = $operatorTypeSpecifyingExtensions;
```

Comment:
You should add support for `operatorTypeSpecifyingExtensions` implementing `BrokerAwareExtension` on line 137.